README.md

```diff
@@ -101,7 +120,7 @@ sh gradlew bootRun
 
 ### Deploy the frontend
 
-To run the front end, run in the [**/frontend**](https://github.com/scalar-labs/indetail/tree/master/Q%26A_application/frontend) folder.
+To run the front end, run in the [**/frontend**](frontend/) folder.
```

Comment:
Is it correct to specify like `frontend/`?

Reply:
> Is it correct to specify like frontend/?

yes, It is working fine.

Modified like this `To run the front end, run in the [**/frontend**](frontend) folder.`
It will help to move to `frontend ` of particular branch.